docker-cloud/builds/automated-testing.md

```diff
@@ -79,9 +79,9 @@ Docker repository, regardless of the Autotest settings.
 
     The following options are available:
 
-    * `Off`: no additional tests. Test commits only to branches that are using Autobuild to build and push images.
-    * `Source Repository`: test commits to all branches of the source code repository, regardless of their Autobuild setting.
-    * `Source Repository & External Pull Requests`: tests commits to all branches of the source code repository, including any pull requests opened against it.
+    * `Off`: No additional test builds. Tests only run if they're configured as hooks in an automated build.
```

Comment:
"autobuilds still run the test phase" <- That's what I'm saying. Should it be "as part of" rather than "as hooks in"?